cl/favorites/models.py

```diff
@@ -67,6 +72,10 @@ class Meta:
         )
 
 
+@pghistory.track(
+    pghistory.Snapshot(),
+    obj_field=pghistory.ObjForeignKey(related_name="history"),
+)
```

Comment:
Since this is a through table without any extra fields, I think we don't need to track it.